src/rules/shopify-lead-from-login.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @title shopify-leads-from-login
+ * @overview Add lead to Shopify at login
+ * @gallery true
+ * @category webhook
+ *
+ * This rule is used to add user accounts to Shopify as user logs in
+ *
+ */
+
+async function addShopifyUser(user, context, callback) {
+  const fetch = require('node-fetch@2.6.0');
+  const { URL } = require('url');
+  const CONFIG = {
+    API_KEY: 'YOUR_API_KEY',
+    API_PWD: 'YOUR_API_PASSWORD',
+    API_URL: 'YOUR_STORE.myshopify.com'
+  };
+  try {
+    const url = new URL(
+      `https://${CONFIG.API_KEY}:${CONFIG.API_PWD}@${CONFIG.API_URL}/admin/api/2020-04/customers.json`
+    );
+    const res = await fetch(url, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        // instead of hard-coding, use Auth0 user object to dynamically build customer data
```

Comment:
I think we should actually show the end-user how to use the Auth0 object here instead of just providing this hardcoded list of properties. What do you think?

Reply:
Like: 
user.app_metadata = user.app_metadata || {};
  if (user.app_metadata.customId) {
    console.log('Found ID!');
    return callback(null, user, context);
or similar?

Reply:
Will do.